Events.lua

```diff
@@ -441,6 +446,7 @@ function WarpDeplete:OnPlayerDead(ev)
   -- good method for deduping though.
   -- self:BroadcastDeath()
   self:ResetCurrentPull()
+  ObjectiveTrackerFrame:Hide()
```

Comment:
```suggestion
  -- Blizzard re-shows the objective tracker every time the player
  -- dies, so we need to re-hide it
  ObjectiveTrackerFrame:Hide()
```

Reply:
I think we would also need to hide the tracker from the other addons that show it again, but to be honest that's not that important to me right now, we can fix that when we get reports that there's an issue.